shell/app-shell/app-shell.js

```diff
@@ -73,15 +83,15 @@ const template = html`
     on-search="_onStateData"
   ></arc-planner>
 
-  <shell-stores
+  <!-- <shell-stores
```

Comment:
Is this intentionally commented out?

Reply:
Yes, shell-stores is deprecated. I'll wait a bit until I'm confident I don't need it, then remove it completely.